packages/fxa-customs-server/package.json

```diff
@@ -20,7 +20,7 @@
     "format": "prettier '**' --write"
   },
   "dependencies": {
-    "@google-cloud/pubsub": "0.24.1",
+    "@google-cloud/pubsub": "^0.31.0",
```

Comment:
The previous version of pubsub was incompatible with node 12 and this update fixed that.